reactor-netty-http/src/main/java/reactor/netty/http/client/Http3ConnectionProvider.java

```diff
@@ -153,6 +154,11 @@ protected InstrumentedPool<Connection> createPool(
 		return new PooledConnectionAllocator(id, name(), parent, config, poolFactory, remoteAddress, resolverGroup).pool;
 	}
 
+	@Override
+	protected ConnectionPoolMetrics delegateConnectionPoolMetrics(InstrumentedPool.PoolMetrics metrics) {
+		return new Http2DelegatingConnectionPoolMetrics((Http2Pool) metrics);
+	}
```

Comment:
Defines the transmission of HTTP/2 metrics in the same way as `Micrometer` configuration for HTTP/3.